addons/html_editor/static/src/main/link/link_popover.xml

```diff
@@ -170,7 +170,13 @@
                         </span>
                         <div class="ms-1 w-100">
                             <div class="d-flex">
-                                <a href="#" target="_blank" t-attf-href="{{state.url}}" class="o_we_url_link fw-bold flex-grow-1 text-truncate" style="max-width: 160px;" t-attf-title="{{state.urlTitle}}">
+                                <a
+                                    href="#"
+                                    t-att-target="linkPreviewTarget"
```

Comment:
I'm a bit confused as to why the other attributes are built from `state.something` but this one is not. Was there a particular reason to exclude this one from the state?